google/cloud/bigtable/README.md

```diff
@@ -23,6 +23,16 @@ is available [online][doxygen-link].
 
 ### v0.4.x - TBD
 
+* More experimental asynchronous APIs, most of the admin operations are now
```

Comment:
These are two separate items, right? Admin operations aren't themselves "experimental" but they're probably implemented using async functionality, so how about:

> * Added more experimental asynchronous APIs
> * Most admin operations are now implemented asynchronously

or similar?

Reply:
Thanks for making the changes!

LGTM, but someone else should review & approve for the overall plan/roadmap, I just had this one comment on wording.